### DISCUSSION TITLE
Release v1.9.0

### DISCUSSION THREAD

**Discussion by aalmiray** [jreleaser/jreleaser#1521] [Announcements]
## Binaries

[https://github.com/jreleaser/jreleaser/wiki/Release-v1.9.0](https://github.com/jreleaser/jreleaser/wiki/Release-v1.9.0)

## 🚀 Features
**core**
- d0c421f Add a f_file_exists template function, closes #1485

## 🐛 Fixes
**assemble**
- 7f09454 Add missing template option bindings to jlink, closes #1496

**changelog**
- 61893b4 Check empty commit messages, closes #1481

## 🧰 Tasks
**build**
- b563e62 Configure Bluesky announcer, closes #1479

## 🛠  Build

- 495a5a8 Update release configuration
- 56f6804 Update release announcements [skip ci]
- 4eafc36 update actions/checkout to v4

## 📝 Documentation

- 2b387cb fix typo in CLA [ci skip]
- 9a93fdc Add jvalkeal as a contributor for code [skip ci]
- 1a00ae8 Add lab596 as a contributor for code

## ⚙️  Dependencies

- 22fb3ba update commons-compress to 1.24.0
- 864886d update zstd-jni to 1.5.5-8
- ce9f4c5 update zstd-jni to 1.5.5-7
- f214bc2 update commons-net to 3.10.0
- 1f04bfe update sshj to 0.37.0
- d4388c3 update mustache to 0.9.11
- 684f68d update jsonschema to 4.32.0
- 6407302 Update feign to 13.0
- ede56c0 update commons-text to 1.11.0
- 2826af2 update maven-annotations to 3.10.1
- c0af285 update tika to 2.9.1
- f18262e update jsoup to 1.16.2
- 4dff901 update aws-java-sdk to 2.21.10
- 32e8368 Update codeql to 2.22.5
- 2cb8332 bump jfrog/frogbot to 2.18.0
- 0e2de9e Update openssf-scorecard to 2.3.1
- e4a487d switch bounctycastle to jdk15to18
- f5bc529 bump bouncycastle to jdk18 & 1.76
- e21b5f3 bump jfrog/frogbot from to 2.12.4
- 5eb82ab Update codeql to 2.21.5
- e5b9267 update junit-pioneer to 2.1.0
- e094b1f Update slf4j to 2.0.9
- 5e7e891 update sshj to 0.36.0

---
- 383c727 Releasing version 1.9.0
- f6c5f50 Update cancel-workflow to 0.12.0
- f7d8a43 Bump for next development cycle

## Contributors
We'd like to thank the following people for their contributions:
- Andres Almiray ([@aalmiray](https://github.com/aalmiray))
- Janne Valkealahti ([@jvalkeal](https://github.com/jvalkeal))
- Matthias Cullmann
- Rohan Udupa ([@lab596](https://github.com/lab596))

<hr /><em>This discussion was created from the release <a href='https://github.com/jreleaser/jreleaser/releases/tag/v1.9.0'>Release v1.9.0</a>.</em>

**Comment by jvalkeal** [comment]
Btw `jreleaser-all-1.9.0.intoto.jsonl` is missing from artifacts.

**Comment by aalmiray** [comment]
Yes, it is. There was a problem during the release on CI and had to post artifacts on a local environment. The consequence is that I can not generate attestation with the current workflow. i’ll have to create a custom workflow just to have the attestation file in place. Will work on that next week

**Reply by jvalkeal** [reply]
Yeah I was curios what your steps are when gh actions run go south. Then noticed that file was missing.

**Reply by aalmiray** [reply]
Matter of fact there’s a blog post coming precisely about this incident and how I was able to push the release after all 😉
